Makefile

```diff
@@ -2,7 +2,7 @@
 
 UNAME_S := $(shell uname -s)
 ifeq ($(UNAME_S), Darwin)
-	LDFLAGS=-framework Foundation
+	LDFLAGS=-framework Foundation -framework Security
```

Comment:
just to check, does this need to be documented in the release readme?

Reply:
Does it? I'm not sure what this does tbh. I got an error on CI and that was the suggested solution I found on the web.

Reply:
It shouldn't, these are mac specific and exist on all macs

Reply:
But also I'd like to see if we can yeet this Makefile at some point because it doesn't seem to do much that isn't already handled by Cargo